### PR TITLE
va-icon: Fixing pill icon's hard-coded fill

### PR DESCRIPTION
**Pull request by Andrew565**
## Chromatic

https://2875-fix-pill-icon-fill--65a6e2ed2314f7b8f98609d8.chromatic.com

---

## Description
Closes [#2875](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2875)

## QA Checklist
- [X] Component maintains 1:1 parity with design mocks
- [X] Text is consistent with what's been provided in the mocks
- [X] Component behaves as expected across breakpoints
- [X] Accessibility expert has signed off on code changes (if applicable. If not applicable provide reason why)
- [X] Designer has signed off on changes (if applicable. If not applicable provide reason why)
- [X] Tab order and focus state work as expected
- [X] Changes have been tested against screen readers (if applicable. If not applicable provide reason why)
- [X] New components are covered by e2e tests; updates to existing components are covered by existing test suite
- [X] Changes have been tested in vets-website using Verdaccio (if applicable. If not applicable provide reason why)

## Screenshots
N/A

## Acceptance criteria
- [X] QA checklist has been completed
- [X] Screenshots have been attached that cover desktop and mobile screens

## Definition of done
- [X] Documentation has been updated, if applicable
- [X] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)

### CODE REVIEW

**Review by jamigibbs — COMMENTED** [review]
Is it just me or does the pill icon look like a lighter gray than the other icons? 

Screenshot 2024-05-28 at 2 05 58 PM

**Comment by Andrew565** [comment]
Chromatic.com is still serving the old sprite sheet, I'll see if I can figure out how to update that.

File: packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts [hunk 1]
```diff
@@ -273,7 +273,7 @@ describe('uswds - va-pagination', () => {
                   <span class="usa-pagination__link-text">next</span>
                   <div id="next-arrow-icon">
                     <svg class="usa-icon" aria-hidden="true" role="img">
-                      <use href="/assets/sprite.svg#navigate_next"></use>
+                      <use href="/img/sprite.svg#navigate_next"></use>
```
Comment (jamigibbs) [inline packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts:276]: This doesn't seem to be related to the pill icon. Is there another issue that we should be linking to for reference?

File: packages/core/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
   "description": "VA.gov component library. Includes React and web components.",
-  "version": "42.1.0",
+  "version": "42.1.1",
```
Comment (jamigibbs) [inline packages/core/package.json:4]: Will a release be made after this is merged? If not, we should revert this and instead just update the web-components package version.

**Review by jamigibbs — APPROVED** [review]

File: packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts (hunk 1 above)
Comment (Andrew565) [reply packages/web-components/src/components/va-pagination/test/va-pagination.e2e.ts:276]: No, it's just a bug I found when simplifying the "where does the sprite sheet go" documentation. We had different copies of the sprite sheet in both the assets folder and img folder, and I was still updating the assets folder version even though it's no-longer being used, only to discover that the va-pagination component had never been updated to look for the img folder version (hope that makes sense?!?). So I updated the va-pagination component to look in the right place and got rid of the old duplicate so no one else makes the same mistake I did and update the wrong sprite sheet.

File: packages/core/package.json (hunk 2 above)
Comment (Andrew565) [reply packages/core/package.json:4]: Can do, not planning a release right away any more.
